README.md

```diff
@@ -62,9 +85,38 @@ yarn run dev
 3.Open <http://localhost:3000> with your browser to see the result.
 You can start editing the page by modifying app/page.tsx. The page auto-updates as you edit the file.
 
+## Choosing an Issue
+
+Choose an unassigned issue from the [Issue's page on Github](https://github.com/distributeaid/next-website-v2/issues) or see which issues are `Ready for Work` in the [DA Project Board](https://github.com/orgs/distributeaid/projects/15). Note: issues labeled `Good first issue` are ideal for newcomers. Once chosen, be sure to assign yourself to the issue and mark the issue as `In Progress`.
+
 ## Branching & Workflows
 
-We'll have a long-running branch(saga) and short-lived branches that will be used to develop parts of the website. An example of a short-lived branch would be feat/about-us. Always ensure you're working on a seperate branch to avoid merge conflicts
+We'll have a long-running branch(**saga**) and short-lived branches that will be used to develop parts of the website. An example of a short-lived branch would be feat/about-us. Always ensure you're working on a seperate branch to avoid merge conflicts.
+
+## Coding Conventions / Best Practices
+
+- Try to avoid using raw #hexadecimal values. View `/src/stylesheets/` or `tailwind.config.ts` to see color value names.
+- Try to use Radix Ui's component library and use tailwind Css for supplimentary styling.
```

Comment:
I agree with Kai about not needing the "Try to" language, and there are a couple of little case inconsistencies and a typo:

```suggestion
- Use Radix UI's component library and use tailwind CSS for supplementary styling.
```